### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by grunweg — COMMENTED** [review]
Thanks!
maintainer merge

File: Mathlib/Analysis/SpecialFunctions/Log/ENNRealLog.lean [hunk 1]
```diff
@@ -51,24 +51,20 @@ noncomputable def log (x : ℝ≥0∞) : EReal :=
...
   · rfl
-  · trivial
-  · rw [ENNReal.toReal_ofReal]
-    exact (not_le.mp h_nonpos).le
+  · rw [ENNReal.toReal_ofReal (not_le.mp h_nonpos).le]
 
 lemma log_ofReal_of_pos {x : ℝ} (hx : 0 < x) : log (ENNReal.ofReal x) = Real.log x := by
-  rw [log_ofReal, if_neg]
-  exact not_le.mpr hx
+  rw [log_ofReal, if_neg (not_le.mpr hx)]
```
Comment (urkud) [inline Mathlib/Analysis/SpecialFunctions/Log/ENNRealLog.lean:60]:
```suggestion
  rw [log_ofReal, if_neg hx.not_le]
```

File: Mathlib/Data/ENat/Basic.lean [hunk 2]
```diff
@@ -45,20 +45,15 @@ instance : WellFoundedLT ℕ∞ := inferInstanceAs (WellFoundedLT (WithTop ℕ))
...
-    rfl
-
 variable {m n : ℕ∞}
 
 /-- Lemmas about `WithTop` expect (and can output) `WithTop.some` but the normal form for coercion
 `ℕ → ℕ∞` is `Nat.cast`. -/
 @[simp] theorem some_eq_coe : (WithTop.some : ℕ → ℕ∞) = Nat.cast := rfl
 
+instance : SuccAddOrder ℕ∞ where
+  succ_eq_add_one x := by cases x <;> simp [SuccOrder.succ]
```
Comment (urkud) [inline Mathlib/Data/ENat/Basic.lean:55]: Unrelated: do we have this for `WithTop`?

**Comment by urkud** [comment]
Thanks!
bors d+

File: Mathlib/Data/ENat/Basic.lean (hunk 2 above)
Comment (Ruben-VandeVelde) [reply Mathlib/Data/ENat/Basic.lean:55]: I'm going to say out of scope :)

**Comment by Ruben-VandeVelde** [comment]
bors merge

File: Mathlib/Probability/Kernel/Composition.lean [hunk 3]
```diff
@@ -916,8 +914,8 @@ lemma fst_map_prod (κ : Kernel α β) {f : β → γ} {g : β → δ} (hg : Mea
     fst (map κ (fun x ↦ (f x, g x))) = map κ f := by
   by_cases hf : Measurable f
   · ext x s hs
-    rw [fst_apply' _ _ hs, map_apply' _ (hf.prod hg), map_apply' _ hf _ hs]
-    · rfl
+    rw [fst_apply' _ _ hs, map_apply' _ (hf.prod hg) _, map_apply' _ hf _ hs]
+    · simp only [Set.preimage, Set.mem_setOf]
```
Comment (YaelDillies) [inline Mathlib/Probability/Kernel/Composition.lean:918]: What's going on here? That's untidying

**Pull request closed without merging**